utils/common.cpp

```diff
@@ -131,6 +131,12 @@ int OpenFile(std::string path, int flag, mode_t mode) {
   return fd;
 }
 
+size_t GetFileSize(int fd) {
+  off_t size = SeekFile(fd, 0, SEEK_END);
+  SeekFile(fd, 0, SEEK_SET);
+  return size;
+}
```

Comment:
This should work in common cases, but as noted in https://www.jpcert.or.jp/sc-rules/c-fio19-c.html, I think actually this is not recommended. So maybe this function should be modified, or otherwise should be renamed (like `GetFileSizeWithSeek`) and moved to other place than utils/"common".cpp

Reply:
Very nice. I fixed Util::GetFileSize() to use fstat(). Fixed